packages/gatsby-react-router-scroll/src/session-storage.ts

```diff
@@ -3,7 +3,7 @@ const STATE_KEY_PREFIX = `@@scroll|`
 const GATSBY_ROUTER_SCROLL_STATE = `___GATSBY_REACT_ROUTER_SCROLL`
 
 export class SessionStorage {
-  read(location: Location, key: string): number {
+  read(location: Location, key: string): number | null {
```

Comment:
where does null come from?

Reply:
huh, i guess it was left over from some development. removing